### ISSUE TITLE
Page disappear from time to time and shown only after maximize the window

### ISSUE THREAD

**Issue by danies8** [dotnet/maui#5967]
### Description

Page disappear from time to time and shown only after maximize the window

### Steps to Reproduce

It difficult for me to give steps.
But it happedned from time to time

### Version with bug

Preview 14 (current)

### Last version that worked well

Preview 14 (current)

### Affected platforms

Windows

### Affected platform versions

....

### Did you find any workaround?

Maximize the window

**Comment by jfversluis** [comment]
I'm really sorry but "this happens from time to time" and no reproduction project makes it impossible to make this issue actionable. Please open a new issue whenever you have something more concrete.
